azure-pipelines.yml

```diff
@@ -133,12 +133,7 @@ jobs:
     projects: $[ dependencies.generator.outputs['mtrx.projects'] ]
   steps:
   - script: |
-      $(JULIA) -e "using BinaryBuilder, Pkg; \
-        name = \"${NAME}\"; \
-        BinaryBuilder.init_jll_package(name, \
-            joinpath(Pkg.devdir(), string(name, \"_jll\")), \
-            string(\"JuliaBinaryWrappers/\", name, \"_jll.jl\") \
-        )"
+      $(JULIA) $(Build.SourcesDirectory)/.ci/jll_init.jl "$(Agent.TempDirectory)/${NAME}.meta.json"
```

Comment:
It looks like it might work:
```
Yggdrasil/S/Sundials@5% julia ../../.ci/jll_init.jl meta.json     
[ Info: Initializing JuliaBinaryWrappers/Sundials_jll.jl...
```

I'm only not 100% sure about the path of the JSON file passed here, but I'm mildly confident that it's correct.